### ISSUE TITLE
[Spell][Druid] Shadowmeld

### ISSUE THREAD

**Issue by NatsumeLily** [Stormforge-gg/Bugtracker-Wotlk-Beta#752]
[//]: # (Enclose links to things related to the bug using http://wowhead.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Zone] A Vision of the Past)
[//]: # ([NPC] Thoralius the Wise)
[//]: # ([Spell][Class] Frostfire Bolt)
[//]: # ([Talent][Class] Brain Freeze)
[//]: # ([Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([Npc][Drop] Thoralius the Wise)
[//]: # ([Web] Armory doesnt work)
**Current behaviour**: Shadowmeld is not resetting bosses when you are the last one alive. 

**Expected behaviour**: Bosses should reset if you use shadowmeld and are the last one alive. 

**Revision**: 1275

**Comment by Godmatik** [comment]
All night elves can shadowmeld, not just druids, might want to edit title.

What's the proof though that shadowmeld can reset bosses? I think that only works for Vanish and Feign Death?

**Comment by FX-SF** [comment]
Test on more than 1 boss to confirm, make sure no one else is there (ie a gm or something).

there appears to be a very intricate hidden detail on how the spell works: In specific, it appears that the Smeld spell applies a temporary threat wipe, but upon leaving shadowmeld, the threat is restored. Theres more to it than that, 

![image](https://user-images.githubusercontent.com/105084838/170632679-e07b5cf4-3f02-415b-9160-e69239adadb7.png)
![image](https://user-images.githubusercontent.com/105084838/170632779-2910e49b-5b8b-4fa8-b8b4-a93bea11c249.png)
![image](https://user-images.githubusercontent.com/105084838/170632863-edcad828-9e33-47e9-ac00-2cb7a15898c8.png)
![image](https://user-images.githubusercontent.com/105084838/170633616-195bf084-a010-41aa-b683-befaef976732.png)

**Comment by Nuy** [comment]
Resetting the boss with Vanish/Shadowmeld isn't a thing and that Wowhead comment proves it.

**Comment by Vishiswaz** [comment]
It IS a thing, as of 3.0.8, read what the comment on 2009/01/18 says.

**Comment by Nuy** [comment]
> It IS a thing, as of 3.0.8, read what the comment on 2009/01/18 says.

Imagine thinking this server will have abilities/talents working like it did in early Wrath. We're on 3.3.5 patch man.

**Comment by Vishiswaz** [comment]
No fuckin shit dude.

Do you have any evidence that Shadowmeld went BACK to working as how it did in 3.0.3 at some point after 3.0.8? Because there isn't any above.

**Comment by Nuy** [comment]
> No fuckin shit dude.
> 
> Do you have any evidence that Shadowmeld went BACK to working as how it did in 3.0.3 at some point after 3.0.8? Because there isn't any above.

What 3.0.8 comment are you refering to?

**Comment by Vishiswaz** [comment]
![image](https://user-images.githubusercontent.com/78314117/170711916-578d3170-a13e-4c9c-a92d-a5446b264dca.png)

**Comment by Nuy** [comment]
Pretty sure bosses or MAYBE even raid instances in general have different rules on this. Boss is not like your regular creature. If indeed you should be able to reset the boss with Shadowmeld then surely same thing should be possible with Vanish.

![opera_OPtzG0kxx6](https://user-images.githubusercontent.com/4800827/170712420-a1eb0cfc-0eb6-4304-9926-a08994272436.png)

**Comment by Vishiswaz** [comment]
The same thing IS possible with Vanish. And Feign Death.

**Comment by Vishiswaz** [comment]
https://flyv.typepad.com/warcraft/2009/01/the-many-uses-of-shadowmeld.html
![image](https://user-images.githubusercontent.com/78314117/170713912-d66b2413-07a8-4b7e-8541-c1f7be31d6f4.png)

**Comment by Vishiswaz** [comment]
I raided alliance during original wotlk. I'm very, very sure this was a thing. I also spent a LOT of time in wintergrasp griefing people (I was a teenager, don't judge too hard) and I would shadowmeld-flight-form near instantly away from them if needed.

There may have been occasions where a few specific boss mechanics may have continued a few seconds after reset and therefore made this fail. But if that's the case, they were the exception to the rule.

**Comment by Nuy** [comment]
I was wrong then and many other servers have it broken. Probably doesn't work correctly for Vanish and maybe Feign Death either  so someone should check it.

**Comment by Vishiswaz** [comment]
Yeah I've been on bugged servers as well with this so I know what you mean.

We tested Feign Death yesterday. [It's actually working TOO well.](https://github.com/Stormforge-gg/Bugtracker-Wotlk-Beta/issues/754)

Haven't tested vanish, I don't think.
